ui/app/AppLayouts/Chat/views/ActivityCenterMessageComponentView.qml

```diff
@@ -253,7 +250,7 @@ Item {
 //                }
 //                return chatType
             }
-            profileImage: realChatType === Constants.chatType.oneToOne ? appMain.getProfileImage(chatId) || ""  : ""
+            profileImage: realChatType === Constants.chatTypeOneToOne ? Global.getProfileImage(chatId) || ""  : ""
```

Comment:
Not sure if this is actually part of this refactor but `chatType.oneToOne` has more of a enum-ish kind of feel to it than `chatTypeOneToOne`.

If this was changed in this PR, any particular reasoning this was done?
(If it was not, ignore this question)